website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -166,6 +166,7 @@ The following table explains the `mesh-task` input variables relevant to a secur
 | `acls`                      | boolean  | If true, ACLs are enabled.                                                                                                                                                                                                                 |
 | `consul_http_addr`          | string   | The Consul server address. Required when `acls = true` in order to log in to Consul's AWS IAM auth method to obtain ACL tokens.                                                                                                            |
 | `consul_https_ca_cert_arn`  | string   | (optional) The Secrets Manager secret containing the CA cert for HTTPS communication with Consul servers. Required if the server's certificate is self-signed or signed by an internal CA. This is not required for Consul servers in HCP. |
+| `audit_logging`             | boolean  | <EnterpriseAlert inline /> (optional) If true, ACL audit logging is enabled.                                                                                                                                                               |
```

Comment:
Here we can also mention that the Consul client is configured to print audit logs to stdout.

Reply:
Suggestion added [here](https://github.com/hashicorp/consul/pull/13729/commits/a25c0c5c9aa345958fd3a0a1bcb0f1d34a00b782)